### ISSUE TITLE
Nominating an item from all content and from the bookmarklet causes the nomination count to equal 0

### ISSUE THREAD

**Issue by regan008** [PressForward/pressforward#998]
Likely related to #985, I am noticing that the following happens: 
1. an item is nominated from all content and the count of nominations is 1
1. the same item is is nominated from the bookmarklet and the count decreases to 0

**Comment by regan008** [comment]
We can discuss it tomorrow but perhaps the best course of action is to push this issue and the fix in #985 to the next milestone.

**Comment by jccatalano** [comment]
This is related to #985 along with other closed issues that #985 is intended to resolve see #960 #961 #962

**Comment by AramZS** [comment]
As a bookmarklet-related error for a fringe case--nominating from feed and then bookmarklet seems much less likely than nominated and then sending to draft from all content--let's push this to be resolved by in 5.2.x.

**Comment by regan008** [comment]
I'm don't think this is a fringe case but lets discuss it today.

**Comment by jccatalano** [comment]
This is the result of a user un-nominating their own nomination. If multiple users nominate a post via different methods, the count is correct. This issue will only cause the count to be off by 1 if the same user nominates via both methods.

**Comment by boonebgorges** [comment]
Previously, there were some cases where the nomination_count postmeta was out of sync. I've made a number of changes to reduce the reliance on this bit of metadata, instead doing just-in-time counts of actual nominators. This eliminates some points of potential confusion.

As for the specific behavior being reported here, I don't think it's so much a bug as an odd bit of interface. Compare:

1. On Feed Items, the 'Nominate' button status is green or grey depending on whether *the current user* has nominated that item. That is, an item could have been nominated by user B, but when user A sees the item on Feed Items, they'll see only a grey Nominate button. If A clicks the button, it turns green. If the button is green (A has already nominated the item), then clicking the button will remove the item-nomination event for that specific user (ie the userA-item pair)
2. On Nominated Items, the 'Nominate' button gives a nomination count that is not user specific. The button is grey if it's never been nominated (though this should never happen on the Nominated Items panel!). It's blue if it has at least one nomination, and it has a nomination count. The button here doesn't actually do anything.

I'm unsure what to make of all this. It doesn't really make sense to have a Nominate *button* when viewing a list of nominated items. Yet the count is a useful piece of info.  On Feed Items, I think it makes sense to have a 'Nominate' button, though TBH I'm not sure we need to support "unnomination" events at all. It *also* makes sense to see a Nomination Count on Feed Items. I think this points toward a separation of concerns: 

- Show a bit of interface on both panels that gives a nomination count, but this interface does not need to be a button in either place. 
- Separately, have a Nominate button that appears only on Feed Items. It can show the current user's nomination status (like it currently does) but it doesn't need to have un-nominate functionality.

I'll let those ideas marinate a bit before implementing.

**Comment by boonebgorges** [comment]
I've made the following changes:

- On 'Nominated Items', I removed the button altogether. Users can see the Nominated Count by clicking the Info button. (Note that this is a rollback of the "greater visibility" referenced in 319e9715eb74276e8b742f935bde20c37bc263d0.)
- On 'Feed Items', when a button is green because you've nominated the item, the tooltip now says 'You have nominated this item' instead of 'Nominate', and the button action is disabled.

This eliminates the main path in the application to produce the behavior originally described in this ticket (where you accidentally unnominate your own post).
